sql/moz-fx-data-shared-prod/telemetry/adm_engagements_daily/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.adm_engagements_daily`
+AS
+SELECT
+  *
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.adm_engagements_daily_v1`
```

Comment:
Looks like a check is failing because this table is not defined yet? @teonbrooks - where are these tables being saved currently?

Reply:
currently there are no tables being saved or written. it's currently only forming a temporary table in redash

Reply:
This check is failing because this table is defined in this PR and not yet created.
I'm not sure though if you need it exposed as a view in `mozdata`?

Reply:
We probably want this saved as a table in `mozdata`. IIUC, creating this as a view in re:dash was a convenient work around.

Reply:
to clarify, i was mentioned the current state in redash. for the view here, i think the issue is that the reference table for the view is being computed in this PR. are we able to create the table and the view in the same PR or do we need to land the table first, and then put the view in afterwards?

Reply:
We'll have table in shared-prod and view in mozdata. This is in line with our convention and will allow for easy access from Redash.

Reply:
SGTM!